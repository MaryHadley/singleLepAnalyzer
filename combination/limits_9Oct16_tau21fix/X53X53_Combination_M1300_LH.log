000 (1295373.000.000) 10/09 22:41:05 Job submitted from host: <192.168.63.249:41533>
...
001 (1295373.000.000) 10/09 22:41:26 Job executing on host: <192.168.63.229:44453>
...
006 (1295373.000.000) 10/09 22:41:34 Image size of job updated: 155380
	46  -  MemoryUsage of job (MB)
	46920  -  ResidentSetSize of job (KB)
...
006 (1295373.000.000) 10/09 22:46:34 Image size of job updated: 282896
	144  -  MemoryUsage of job (MB)
	147384  -  ResidentSetSize of job (KB)
...
006 (1295373.000.000) 10/10 00:01:42 Image size of job updated: 282896
	144  -  MemoryUsage of job (MB)
	147396  -  ResidentSetSize of job (KB)
...
006 (1295373.000.000) 10/10 17:23:30 Image size of job updated: 297440
	157  -  MemoryUsage of job (MB)
	160368  -  ResidentSetSize of job (KB)
...
005 (1295373.000.000) 10/10 17:29:39 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 18:46:39, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 18:46:39, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	1596  -  Run Bytes Sent By Job
	307  -  Run Bytes Received By Job
	1596  -  Total Bytes Sent By Job
	307  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :    31817        1   3824812
	   Memory (MB)          :      157     3072      3072
...
