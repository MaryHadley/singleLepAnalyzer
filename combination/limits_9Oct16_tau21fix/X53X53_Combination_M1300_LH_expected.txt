# x; y; band 0 low; band 0 high; band 1 low; band 1 high
         0   0.079026    0.03999    0.16335  0.054827    0.11601
